otherlibs/runtime_events/runtime_events_consumer.c

```diff
@@ -345,19 +354,31 @@ void caml_runtime_events_free_cursor(struct caml_runtime_events_cursor *cursor){
   if (cursor->cursor_open) {
     cursor->cursor_open = 0;
 #ifdef _WIN32
-    UnmapViewOfFile(cursor->metadata);
+    UnmapViewOfFile(cursor->map);
     CloseHandle(cursor->ring_file_handle);
     CloseHandle(cursor->ring_handle);
 #else
     /* This cast is necessary for compatibility with Illumos' non-POSIX
       mmap/munmap */
-    munmap((void*)cursor->metadata, cursor->ring_file_size_bytes);
+    munmap((void*)cursor->map, cursor->ring_file_size_bytes);
```

Comment:
The comment and the cast might also be unnecessary now, see [illumos `munmap`](https://illumos.org/man/2/munmap).